outline.md

```diff
@@ -0,0 +1,59 @@
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
+To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've begun to pursue those goals.
+Even though the learning in this presentation is occurring in that context, we suspect that much of what's here applies generally.
+
+When building an inner source program you should have a vision of what services you will provide to facilitate inner sourcing _as well as_ a strategy you will employ to roll out and make effective these services in your organization.
+We'll explain principles, resources, and specific examples that can aid and guide in doing so.
+
+The services of the inner source program should provide two things.
+First, they must teach, define, and document the type of culture that is conducive to inner source.
+Second, there must be software that reinforces that culture while simultaneously lowering the barrier to entry to participating in that culture.
+Examples of these services include:
+
+* Documentation on how to do inner source (and relation of that documentation to [InnerSourcePatterns](http://innersourcecommons.org/patterns)).
+* Software for automated, two-way discovery of inner source projects (maintainer to contributor/user and contributor/user to maintainer).
+* Software for a data-driven understanding of the impact of inner source projects.
+* Software/convention to facilitate individuals communicating with one another.
+* Curated training on the above software and documentation.
+
+The rollout of an inner source program is similar to the rollout of any large, grass-roots initiative.
+Documented patterns in this area (well-summarized in [the Fearless Change series](http://www.fearlesschangepatterns.com/)) apply and work.
+We'll review real examples of how these patterns played out in our journey.
+We'll also share a few patterns unique to inner source rollout including:
+
+* Run your inner source program as inner source projects.
+* Do (or say) nothing save it is documented.
+* Program growth/maturity progression starts with facilitating any synchronous, in-person, side-project talking collaboration then grows to facilitating asynchronous, main-project coding collaboration.
+
+After explaining these principles with clear words and real anecdotes, we'll summarize the current state of our inner source program, where we're hoping it will go, and how we aim to get there. We'll express our hope to learn from and work with everyone present via open dialogue and sharing in the inner source commons.
+
+#### Takeaways
+
+After attending this presentation, we hope that viewers will:
+
+* Understand the suggested purposes and principles of an inner source program.
+* Understand the suggested component parts of an inner source program and their principles.
```

Comment:
This might be too much to go into?  I'd rather you focus on clearly articulating "suggested purposes and principles" than getting into the weeds on the things that fall out from that e.g. "component parts" 

I'm saying I think it is ok to share ideas and what we've started doing but I don't think that should be the emphasis of your talk.  So rather than having them understand this as a result of your talk, I'd rather see them have a framework for talking about potential component parts so that proposals for atomic parts can begin to surface in the community as we have begun to do with our value stream work.

This will lead to more collaboration for the last two takeaways in your list.

Reply:
The goal is definitely more collaboration, I guess I thought that they may need to know some specifics on what we're doing to be interested in collaborating on it.  Do you feel they'd get enough motivation just by understanding the general principles we're trying to follow?

I feel like we need to seed the idea that there are some specific projects that we all will need in order to get everyone's brains going on the idea of working on it together.

Reply:
I like the measurable goal of this presentation as actual collaboration with other teams in other companies on our work. That's definitely something we can measure! It's also a really exciting idea! Excited to see the presentation now with this goal in mind!